### PR TITLE
[ToggleGroup] Improve `orientation` prop description to clarify function

### LINKED ISSUE #463: Consider being more precise in terms of what the `orientation` prop does (ie. no visual changes).
### Current Behavior

The orientation prop has no effect on `ToggleGroup`. No matter what, the ToggleGroup has a horizontal orientation.

### Expected behavior

Similar to the Slider, `orientation` ought to change the ToggleGroup's orientation; otherwise it should be removed as an option

### Reproducible example

**Note** The following link comes directly from the ToggleGroup [docs](https://www.radix-ui.com/docs/primitives/components/toggle-group). I am also seeing this issue when running locally.

[CodeSandbox Template](https://codesandbox.io/s/kvb8uo?module=App.js&file=/App.js)

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Radix Package(s) |     @radix-ui/react-toggle-group   |     1.0.0    |
| React            | n/a     |     18.0.0    |
| Browser          |  Chrome     |     105.0.5195.52    |
| Assistive tech   |         |         |
| Node             | n/a     |     16.15.0    |
| npm/yarn         |     pnpm    |  6.32.13       |
| Operating System |    macOS     |     Monterey 12.5.1     |

### PR DESCRIPTION
**Pull request by ItzaMi**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code.
- [x] Include the URL where we can test the change in the body of your PR.

This pull request:

Closes #463 

- [ ] Fixes a bug
- [ ] Adds additional features/functionality
- [x] Updates documentation or example code
- [ ] Other

**Before**
Screenshot 2022-10-28 at 18 14 56

**After**
Screenshot 2022-10-28 at 18 22 35

### CODE REVIEW

**Review by benoitgrelard — COMMENTED** [review]
Also once the copy change is made, could you update it in the previous version files for toggle-group too?

Thanks 🙏

File: data/primitives/components/toggle-group/1.0.1.mdx [hunk 1]
```diff
@@ -182,7 +182,13 @@ Contains all the parts of a toggle group.
       type: '"horizontal" | "vertical" | undefined',
       typeSimple: 'enum',
       default: 'undefined',
-      description: 'The orientation of the component.',
+      description: (
+        <span>
+          The orientation of the component, determining move focus within the
```
[the remark this answers is not in the document]
Comment (ItzaMi) [reply data/primitives/components/toggle-group/1.0.1.mdx]: My newest commits changed the copy as requested and added the same info to the previous versions 👍

**Review by benoitgrelard — APPROVED** [review]

**Comment by benoitgrelard** [comment]
Thanks for your contribution @ItzaMi!

**Comment by ItzaMi** [comment]
Glad to help. Thank you!

**Comment by pedrovrima** [comment]
Is this the same case for all orientations? E.g. The navigation-menu?

**Comment by amazfs** [comment]
how to switch arrow navigation from right to left? e.g. if the menu is on the right

**Comment by amazfs** [comment]
when menu is on the right arrow navigation direction should be to the right instead of left. How to fix? 
I setted orientation as 'rtl' but it doesn't work

**Comment by amazfs** [comment]
is there a way to fix arrow navigation direction for bottom menu too? from bottom to up

**Pull request merged**